module/stepfunctions_send_message.tf

```diff
@@ -81,6 +81,7 @@ resource "aws_sfn_state_machine" "send_message" {
           {
             BackoffRate = 2
             ErrorEquals = [
+              "HTTPError",
```

Comment:
this is a tricky one ... the mesh-client specifically avoids retries on the initial send message to avoid creating duplicate messages .. everything but the initial send message is considered retryable ..

I think I'd like to see:
- this as an optional configuration (default to current behaviour)
- support for more discrete http error selection .. ( 400x status codes are not retryable ..and should not be re-attemped)

Reply:
All very good points. Agreed, we'd need to action them before this PR could go in. As discussed on Slack, the first would be trivial enough but the second may involve some code and custom exceptions based on the HTTP Code, as Step function error handling/retries only seems to be able to respond based on the exception type: https://docs.aws.amazon.com/step-functions/latest/dg/tutorial-handling-error-conditions.html - will discuss with the MNS team regarding the priority and potential options.